internal/app/service/achievement_service.go

```diff
@@ -43,3 +45,19 @@ func (achSvc *service) CreateAchievement(ctx context.Context, cDetail dto.Create
 
 	return cDetail.ProfileID, nil
 }
+
+func (achSvc *service) GetAchievements(ctx context.Context, profileID string) (value []dto.AchievementResponse, err error) {
+	ID, err := helpers.ConvertStringToInt(profileID)
```

Comment:
handle it into handler level string to int convert

Reply:
done